src/App.css

```diff
@@ -0,0 +1,167 @@
+body {
+  height: 100%;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  background: linear-gradient(to right, #2376ae, #516176);
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0 20px;
+}
+
+.app-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  height: 100vh;
+  margin: 10% auto;
+}
+
+.search-form-container {
+  text-align: center;
+  margin: 5% auto;
+}
+
+.header {
+  text-align: center;
+  margin: 40px 0;
+}
+
+.header-text {
+  color: #fff;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin-top: 40px;
+  position: relative;
+}
+
+.search-form {
+  display: flex;
+  align-items: center;
+  max-width: 800px;
+  margin: 50px;
+  position: relative;
+}
+
+.search-input {
+  padding: 8px 12px;
+  height: 36px;
+  width: 500px;
+  max-width: 100%;
+  font-size: 16px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+}
+
+.search-label {
+  position: absolute;
+  top: 50%;
+  left: 15px;
+  transform: translateY(-50%);
+  color: #999;
+  font-size: 14px;
+  transition: opacity 0.3s ease;
+  pointer-events: none;
+  opacity: 1;
+}
+
+.search-input:not(:placeholder-shown)+.search-label {
+  opacity: 0;
+}
+
+.search-icon {
+  position: absolute;
+  top: 50%;
+  right: 10px;
+  transform: translateY(-50%);
+  color: #666;
+  cursor: pointer;
+}
+
+.vehicle-specification {
+  background-color: #f8f8f8;
+  padding: 20px;
+  border-radius: 5px;
+  margin-top: 40px;
+}
+
+.spec-list {
+  list-style-type: none;
+  padding: 0;
+  margin: 0;
+}
+
+.spec-item {
+  margin-bottom: 10px;
+}
+
+@media (max-width: 768px) {
```

Comment:
Only suggestion i have is to maybe add another media query for smaller screens, like mobile phones (max-width: 480px & smaller) and to make sure the input box is responsive at this size. Other than that, cool project and great code overall!!